File: app/src/main/java/com/datarockets/mnchkn/store/ScoreService.java
package com.datarockets.mnchkn.store;

public interface ScoreService {
    /**
     *
     * @param score - a player's score
     * @return updated increased score
     */
    int increaseScore(int score);
    /**
     *
     * @param score - a player's score
     * @return updated decreased score
     */
    int decreaseScore(int score);
}

File: app/src/main/kotlin/com/datarockets/mnchkn/store/ScoreService.kt
package com.datarockets.mnchkn.store

interface ScoreService {
    /**

     * @param score - a player's score
     * *
     * @return updated increased score
     */
    fun increaseScore(score: Int): Int

    /**

     * @param score - a player's score
     * *
     * @return updated decreased score
     */
    fun decreaseScore(score: Int): Int
}
